frontend/lib/src/services/user_service.dart

```diff
@@ -17,4 +20,31 @@ class UserService {
       return [];
     }
   }
+
+  static Future<dynamic> deleteUser(String id) async {
```

Comment:
```suggestion
  static Future<void> deleteUser(String id) async {
```
function doesnt have any return